lang/en/texts/mobile-app-open-food-facts.html

```diff
@@ -14,6 +14,8 @@ <h1>Install the Open Food Facts mobile app</h1>
 
 <p>You can also add new products to Open Food Facts and build a common good to improve nutrition and health for everybody. Thank you !</p>
 
+<p>If you use the app and enjoy the Open Food Facts project, it would be nice to leave a review on the App Store or the Play Store to encourage others to install it. Thank you !</p>
```

Comment:
```suggestion
<p>If you use the app and enjoy the Open Food Facts project, it would be nice to leave a review on the App Store or the Play Store to encourage others to install it. Thank you!</p>
```
Unnecessary space